projects/kit/components/combo-box/test-stringify-update.spec.ts

```diff
@@ -0,0 +1,79 @@
+import {ChangeDetectionStrategy, Component, signal} from '@angular/core';
+import type {ComponentFixture} from '@angular/core/testing';
+import {TestBed} from '@angular/core/testing';
+import {FormControl, ReactiveFormsModule} from '@angular/forms';
+import type {TuiStringHandler} from '@taiga-ui/cdk';
+import {TuiTextfield} from '@taiga-ui/core';
+import {NG_EVENT_PLUGINS} from '@taiga-ui/event-plugins';
+import {TuiComboBox} from '@taiga-ui/kit';
+
+interface TestItem {
+    readonly id: number;
+    readonly name: string;
+}
+
+@Component({
+    standalone: true,
+    imports: [ReactiveFormsModule, TuiComboBox, TuiTextfield],
+    template: `
+        <tui-textfield [stringify]="stringify()">
+            <input
+                tuiComboBox
+                [formControl]="control"
+            />
+        </tui-textfield>
+    `,
+    changeDetection: ChangeDetectionStrategy.OnPush,
+})
+class TestComponent {
+    private readonly items = signal<readonly TestItem[]>([]);
+
+    protected readonly control = new FormControl<number | null>(777);
+
+    protected readonly stringify = signal<TuiStringHandler<number>>(
+        (id: number) => this.items().find((item) => item.id === id)?.name ?? String(id),
+    );
+
+    public updateItems(items: readonly TestItem[]): void {
+        this.items.set(items);
+        // Update stringify signal to use new items
+        this.stringify.set(
```

Comment:
Use `computed` for `stringify` (with `items` as dep).

No need to update both in the method